docs/list.json

```diff
@@ -373,6 +373,7 @@
 				"OBJLoader": "examples/en/loaders/OBJLoader",
 				"PCDLoader": "examples/en/loaders/PCDLoader",
 				"PDBLoader": "examples/en/loaders/PDBLoader",
+				"PRWMLoader": "examples/en/loaders/PRWMLoader",
```

Comment:
This entry has been removed. Probably best to rebase.

Reply:
sure, will do that now.